docs/contributing/control-plane/development.md

```diff
@@ -0,0 +1,103 @@
+# Development Guide
+This document is intended to provide contributors with an introduction to developing the control plane of the Sedna project. 
+There are two components in our control plane: [GM(Global Manager) and LC(Local Controller)][framework].
+
+## Overview
+
+Sedna provides various utilities for development wrapped in `make`.
+
+Most scripts require more or less only `make` + `bash` on the host, and generally
+stick to POSIX utilities. Some scripts use `docker` e.g. for image building or
+to use docker images containing special tools.
+
+## Read Conventions Before Coding
+- [k8s coding convention]
+- [k8s api convention] if you want to add or update api.
+
+## Building
+Before continuing, You need to follow the [prerequisites installation guide] if you haven't done it yet.
```

Comment:
follow for what ?

Reply:
[prerequisites installation guide] tells what